frontend/src/metabase/qb/lib/actions.js

```diff
@@ -40,6 +40,11 @@ export const toUnderlyingRecords = (card: CardObject): ?CardObject => {
     }
 };
 
+export const nestThisQuery = (card: CardObject): ?CardObject => {
+    const newCard = startNewCard("query", -1337, "card__" + card.id);
```

Comment:
the whole idea around making the Saved Questions DB look just like a real DB was so we wouldn't have to do things like hardcode magic numbers like this.

At the very least make that magic number a const like `SAVED_QUESTIONS_DB_ID` or something. Better yet look at the DBs in the store and `.find` the one that has `.is_saved_questions === true` and use its ID